.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
```

Comment:
Does this enable dependabot for all deps? I would prefer to have this be security-only.

Reply:
I've updated the config so that it'll be security-only, [according to this documentation](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates):

> If you only require security updates and want to exclude version updates, you can set open-pull-requests-limit to 0 in order to prevent version updates for a given package-ecosystem. For more information, see "[open-pull-requests-limit](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit)."